third_party/bigframes_vendored/pandas/core/generic.py

```diff
@@ -167,7 +167,7 @@ def astype(self, dtype):
             dtype (str or pandas.ExtensionDtype):
                 A dtype supported by BigQuery DataFrame include ``'boolean'``, ``'Float64'``, ``'Int64'``,
                 ``'int64[pyarrow]'``, ``'string'``, ``'string[pyarrow]'``, ``'timestamp[us, tz=UTC][pyarrow]'``,
-                ``'timestamp\[us\]\[pyarrow\]'``, ``'date32\[day\]\[pyarrow\]'``, ``'time64\[us\]\[pyarrow\]'``.
+                ``'timestamp[us][pyarrow]'``, ``'date32[day][pyarrow]'``, ``'time64[us][pyarrow]'``.
```

Comment:
There is a bug in the doc generation, which it renders incorrectly without the escape symbol: https://screenshot.googleplex.com/45wRMR3yQUuuGdB

With the escape symbol: https://screenshot.googleplex.com/7uGCXAoC6coMRoC

Reply:
Bummer. That's really annoying. `\[` is invalid in a Python string, but equivalent to `\\[`. I'll update the text.

Reply:
Fixed in https://github.com/googleapis/python-bigquery-dataframes/pull/572/commits/1e438d25bc71c446aa5b508223fba123957c13f1